hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx

```diff
@@ -65,10 +66,11 @@ const withTaskListContext = (WrappedComponent) => {
     search = debounce(this.updateSubscription, 350);
 
     onSearch = ({ target }) => {
-      const query = target.value.trim();
+      const { value: query } = target;
+      const searchQuery = query.trim();
 
       this.setState({
-        searchQuery: query.length > 2 ? query : '',
+        searchQuery: searchQuery.length > 2 ? searchQuery : '',
         query,
```

Comment:
May be updated query too?
To avoid cases like this.
![image](https://user-images.githubusercontent.com/33573380/66757459-e9702980-eea4-11e9-903f-b92e7271c9cb.png)